src/ui/components/NetworkIcon/NetworkIcon.tsx

```diff
@@ -43,25 +35,22 @@ function TextFallback({
         ...style,
       }}
     >
-      {(!value || value.length > 4) && name
-        ? name.charAt(0).toUpperCase()
-        : value || '?'}
+      {name ? name.slice(0, 3).toUpperCase() : '???'}
```

Comment:
isn't chainId a better fallback than `???`?

Reply:
Both fallbacks are pretty much useless in my opinion

Reply:
`chainId` is a better fallback because two chains with missing names will at least be displayed differently
Adding `title` if the name is missing is a great idea, let's do it

Reply:
But it will require to move all code back :(

Reply:
Also it looks like this fallback can work only if there is no network at all

Reply:
So in this case ??? is even better. 
`name` is not nullable field of `NetworkConfig`

Reply:
Looks like it is almost impossible to have a network with an empty name. But even in that rarest (probably impossible?) case, showing the fallback in the form `???` won't be too bad